src/bsoncxx/json.hpp

```diff
@@ -36,28 +36,6 @@ BSONCXX_INLINE_NAMESPACE_BEGIN
 BSONCXX_API std::string BSONCXX_CALL to_json(document::view view);
 
 ///
-/// Converts an element (key-value pair) to a JSON key-value pair.
-/// This is not actually a valid JSON document, but it can be useful for debugging.
-///
-/// @param element
-///   A BSON element.
-///
-/// @returns A JSON key-value pair.
-///
-BSONCXX_API std::string BSONCXX_CALL to_json(document::element element);
```

Comment:
Ideally, given a doc ```doc``` that would be as simple as

```
bsoncxx::document::element elt = ...;
auto elt_jas_json = to_json(builder::stream::document << elt << finalize);
```

But, interestingly, there doesn't seem to be an stream/basic/core overload for the builders that takes an element. Or am I just overlooking it somehow? @hanumantmk?